lib/features/context-pad/ContextPad.js

```diff
@@ -106,10 +108,30 @@ ContextPad.prototype._init = function() {
 /**
  * Register a provider with the context pad
  *
+ * @param  {Number} [priority=1000]
  * @param  {ContextPadProvider} provider
+ *
+ * @example
+ * contextPad.registerProvider(800, {
+  *   getContextPadEntries: function(element) {
+  *     return function(entries) {
+  *       return Object.assign(entries, {
```

Comment:
This example is dangerous, as it patches `entries` in place. 

Please use ES6 spread operators here:

```
return {
  ...entries,
  'entry-1': { ... }
};
```

Reply:
OK, I'll update both `ContextPad` and `Palette` examples.